qf_lib/common/tickers/tickers.py

```diff
@@ -177,12 +188,15 @@ class BinanceTicker(Ticker):
     point_value:
         size of the contract as given by the ticker's Data Provider. Used mostly by tickers of security_type FUTURE and
         by default equals 1.
+    rounding_precision:
+        rounding precision. Default 5.
     """
-    def __init__(self, currency: str, quote_ccy: str, security_type: SecurityType = SecurityType.CRYPTO,
+    def __init__(self, crypto_currency: str, quote_ccy: str, security_type: SecurityType = SecurityType.CRYPTO,
```

Comment:
As the other parameter is called quote_ccy we could call this one base_ccy

Reply:
Here the `crypto_currency` is the `identifier of the security, e.g. 'BTC'`, so I would hesitate to call it `base_ccy` because the ticker does not really represent a currency conversion